From de6b906bd93e0928cd5464418091268aa7a5ac78 Mon Sep 17 00:00:00 2001
From: Corey Swenson <cswenson@us.ibm.com>
Date: Fri, 27 Feb 2015 09:45:47 -0600
Subject: [PATCH 1/3] Disable SPD writes - workaround for DRAM repairs error

Change-Id: Ic5fac400abff8fef004268dfbdf33fa6ecdded1c
(cherry picked from commit 1bc5bb42be8e81b525993e9b5dcf08fe684bade4)
---
 src/usr/vpd/spd.C |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/usr/vpd/spd.C b/src/usr/vpd/spd.C
index 54837a5..e101ff0 100644
--- a/src/usr/vpd/spd.C
+++ b/src/usr/vpd/spd.C
@@ -323,6 +323,7 @@ errlHndl_t spdWriteKeywordValue ( DeviceFW::OperationType i_opType,
                                   va_list i_args )
 {
     errlHndl_t err = NULL;
+    return err;
     VPD::vpdKeyword keyword = va_arg( i_args, uint64_t );
     VPD::vpdCmdTarget location =
             (VPD::vpdCmdTarget)va_arg( i_args, uint64_t );
-- 
1.7.4.1

